docs/guides/UltimateAi.md

```diff
@@ -0,0 +1,35 @@
+#ultimate.ai setup
+
+1 - ultimate.ai for now does not have a self service account creation. If you don't have an account through having been in contact with ultimate.ai, contact ultimate.ai with email at support@ultimate.ai
```

Comment:
@ultimake , this is amazing! thanks for the contribution. Any change you can create an account for jrico@salesforce.com for me to try things things out?